lib/ssh/src/ssh_connection_handler.erl

```diff
@@ -401,6 +401,7 @@ alg(ConnectionHandler) ->
 %% Intitialisation
 %%====================================================================
 init([Role, Socket, Opts]) when Role==client ; Role==server ->
+    ssh_lib:set_label(Role, {connection, Socket}),
```

Comment:
Move this label to server case? Otherwise you will overwrite it in client case!

Reply:
Yes. I thought that kind of behavior will be fine. Maybe not optimal, since label will be set twice for client ...